app/controllers/todos_controller.rb

```diff
@@ -6,39 +6,48 @@ class TodosController < ApplicationController
 
     # GET /todo_lists/:todos_list_id/todos
   def index
+    authorize @todo_list, :show?
+
```

Comment:
why show?

Reply:
to show the todolist associated with that particular user either his own or collabrative